src/Pretzel.Logic/Templating/JekyllEngineBase.cs

```diff
@@ -46,7 +46,7 @@ public abstract class JekyllEngineBase : ISiteEngine
 
         protected abstract string RenderTemplate(string content, PageContext pageData);
 
-        public void Process(SiteContext siteContext, bool skipFileOnError = false)
+        public void Process(SiteContext context, bool skipFileOnError = false)
```

Comment:
Is this renaming part of a warning fix?

Reply:
warning fix: base class uses `protected abstract string RenderTemplate(string content, PageContext pageData);` hence inherited classes should use the same name.